### PR TITLE
[android] - only dispatch events if not handled by MarkerView

### LINKED ISSUE #8444: MarkerView clicked
Using Mapbox Android SDK
Mapbox SDK version: 5.0.0-beta3

Steps to trigger behavior

when I click a markerview, the onMarkerClick and onMapClick method both are called。

Expected behavior

Only the onMarkerClick method is called；

i don't konw it is a bug or not.I just want distinguish the map or markerview is clicked，Is there any good way to do it?

### PR DESCRIPTION
**Pull request by tobrun**
Closes #8444

### CODE REVIEW

**Comment by tobrun** [comment]
Retesting this and we actually need to be able to handle touches that are handled by the child views.. this makes starting a pan on a MarkerView impossible.

**Comment by tobrun** [comment]
Will need to re implement the OnClick logic with OnTouch to be able to control the flag returned to MarkerViewContainer.

**Comment by tobrun** [comment]
8f38a77 fixes the issue, want to review @Guardiola31337?

File: platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/MarkerViewManager.java [hunk 1]
```diff
@@ -503,18 +504,24 @@ public void invalidateViewMarkersInVisibleRegion() {
...
-                    clickHandled = onMarkerViewClickListener.onMarkerClick(marker, v, adapter);
-                  }
+              adaptedView.setOnTouchListener(new View.OnTouchListener() {
 
-                  if (!clickHandled) {
-                    ensureInfoWindowOffset(marker);
-                    select(marker, v, adapter);
+                @Override
+                public boolean onTouch(View v, MotionEvent event) {
+                  switch (event.getAction()) {
```
Comment (Guardiola31337) [inline platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/MarkerViewManager.java]: Why using `switch` instead `if`? There is only one case (`MotionEvent.ACTION_UP`).

Comment (tobrun) [inline platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/MarkerViewManager.java]: Comment fixed in #439f9f6

**Pull request merged**